src/Documentation/tutorials_and_samples/tutorial_1_helloworld.md

```diff
@@ -0,0 +1,142 @@
+---
+layout: page
+title: Tutorial 1 Hello World
+---
+
+# Tutorial 1: Hello World
+
+This process ties into the Hello World sample application available [here](https://github.com/dotnet/orleans/tree/master/Samples/2.0/HelloWorld).
+The important concepts to get out of this tutorial are:
+Orleans is not a single, standalone application that you download and install.
+Orleans is a collection of separate, configurable projects that use NuGet packages to add Orleans functionality to another application.
```

Comment:
this isn't accurate, Orleans provides nugets to install orleans to your application. Orleans aren't projects per se.

Reply:
Since line 11 was further explanation for line 10, I removed it.